README.md

```diff
@@ -50,6 +50,8 @@ Values are read from environment in [vars/main.yml](vars/main.yml)
 - `NEW_RELIC_ACCOUNT_ID`
 - `NEW_RELIC_REGION`
 
+Additionally, an optional `https_proxy` variable can be set to enable a proxy for your installation. Add it to the `environment` keyword in your `playbook`. See [ansible's remote environment](https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_environment.html) for more info.
```

Comment:
Just curious, isn't the env variable case sensitive? Shouldn't it be HTTPS_PROXY to work?
@amolero-nr

Reply:
My fault @Julien4218, it is indeed case sensitive

Reply:
Thank you @rthorn-nr!